test/powershell/engine/BigIntegerCultureHandling.test.ps1

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+Describe 'PowerShell Type Conversion - BigInteger Parsing' -Tag 'CI' {
+
+    It 'Can convert formatted numbers using PowerShell type system' {
```

Comment:
Thanks for updating. I'm thinking we don't need this and the "hi-IN" test: the new "de-DE" test is sufficient to demonstrate that the behavior is culture-invariant, as desired.